### PR TITLE
Update complaints

### LINKED ISSUE #161: Перетворити вимогу в скаргу: `Resolving variable '${claim.access.token}' failed: AttributeError: access`
_OpenUA - "Можливість подати скаргу на умови більше ніж за 4 дні до завершення періоду подання пропозицій"_

В kw `Перетворити вимогу в скаргу` ожидается, что будет получен параметр `${claim.access.token}`, но драйвер нашей площадки такими данными не обладает. Не могу ее вернуть из `Створити вимогу`.

`Resolving variable '${claim.access.token}' failed: AttributeError: access`

### PR DESCRIPTION
**Pull request by mykhaly**
Closes #161.

---

This change is [Reviewable](https://reviewable.io/reviews/openprocurement/robot_tests/131)

### CODE REVIEW

File: op_robot_tests/tests_files/keywords.robot [hunk 1]
```diff
@@ -410,6 +410,33 @@ Get Broker Property By Username
   [return]  ${field_value}
 
 
+Звірити поле скарги із значенням
+  [Arguments]  ${username}  ${given_value}  ${field_name}  ${complaintID}
+  ${received_value}=  Отримати дані із скарги на умови  ${username}  ${complaintID}  ${field_name}  ${given_value}
+  Порівняти об'єкти  ${given_value}  ${received_value}
+
+
+Отримати дані із скарги на умови
```
Comment (Leits) [inline op_robot_tests/tests_files/keywords.robot]: Чому саме скарги на умови?

File: op_robot_tests/tests_files/keywords.robot [hunk 2]
```diff
@@ -410,6 +410,33 @@ Get Broker Property By Username
...
+  Run keyword And Return if  '${field_name}' == 'document.title'  Отримати заголовок документа
+  ...      ${username}
+  ...      ${complaints[${complaint_index}].documents}
+  ...      ${given_value}
+  ${field_value}=  Get Variable Value  ${USERS.users['${username}'].tender_data.data.complaints[${complaint_index}]['${field_name}']}
+  Log  ${field_value}
+  [Return]  ${field_value}
+
+
+Отримати заголовок документа
```
Comment (Leits) [inline op_robot_tests/tests_files/keywords.robot]: Чому б не "Отримати інформацію із документа" і передавати сюди поле, яке потрібно отримати?

File: op_robot_tests/tests_files/keywords.robot (hunk 1 above)
Comment (mykhaly) [inline op_robot_tests/tests_files/keywords.robot]: бо це скарга на умови закупівлі

File: op_robot_tests/tests_files/keywords.robot (hunk 2 above)
Comment (mykhaly) [inline op_robot_tests/tests_files/keywords.robot]: бо ми в документі нічого крім заголовку не можемо перевірити

Comment (Leits) [inline op_robot_tests/tests_files/keywords.robot]: http://api-docs.openprocurement.org/uk_UA/latest/standard/document.html
Усі можемо перевірити.

File: op_robot_tests/tests_files/keywords.robot (hunk 1 above)
Comment (Leits) [inline op_robot_tests/tests_files/keywords.robot]: Вона чимось відрізняється від інших видів скарг?

File: op_robot_tests/tests_files/keywords.robot (hunk 2 above)
Comment (mykhaly) [inline op_robot_tests/tests_files/keywords.robot]: Окей, зараз спробую переробити.

File: op_robot_tests/tests_files/keywords.robot (hunk 1 above)
Comment (mykhaly) [inline op_robot_tests/tests_files/keywords.robot]: Скарги можуть знаходиться в різних локаціях. Скарга на тендер - tender.data.complaints, скарга на скасування - tender.data.cancellations.complaints. Я це так зрозумів

File: op_robot_tests/tests_files/keywords.robot (hunk 2 above)
Comment (mykhaly) [inline op_robot_tests/tests_files/keywords.robot]: Переробив

**Comment by mykhaly** [comment]
Закривати цей пр і перекидати його на master?

**Comment by Leits** [comment]
Reviewed 2 of 7 files at r2.
Review status: 0 of 7 files reviewed at latest revision, 3 unresolved discussions.

---

*[op_robot_tests/tests_files/aboveThreshold.robot, line 8 \[r3\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGMZFTX1nHo3-BYcZFB:-KGMZFTX1nHo3-BYcZFC:-1356905025) ([raw file](https://github.com/openprocurement/robot_tests/blob/1bd524e5631fc0bd116d15cf9fc62d07b5047f82/op_robot_tests/tests_files/aboveThreshold.robot#L8)):*
openeu

---

*[op_robot_tests/tests_files/aboveThreshold.robot, line 18 \[r3\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGMZGJVL1ViDSuwZKWR:-KGMZGJVL1ViDSuwZKWS:-1681784080) ([raw file](https://github.com/openprocurement/robot_tests/blob/1bd524e5631fc0bd116d15cf9fc62d07b5047f82/op_robot_tests/tests_files/aboveThreshold.robot#L18)):*
delete

---

*[op_robot_tests/tests_files/aboveThreshold.robot, line 24 \[r3\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGMZIjSK_nY4DEvDTol:-KGMZIjSK_nY4DEvDTom:-2079241293) ([raw file](https://github.com/openprocurement/robot_tests/blob/1bd524e5631fc0bd116d15cf9fc62d07b5047f82/op_robot_tests/tests_files/aboveThreshold.robot#L24)):*
Передививсь, що наробив ребейс.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/robot_tests/131)*

**Comment by mykhaly** [comment]
Review status: 0 of 7 files reviewed at latest revision, 3 unresolved discussions.

---

*[op_robot_tests/tests_files/aboveThreshold.robot, line 8 \[r3\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGMZFTX1nHo3-BYcZFB:-KGMrVHbR0cB69vXZDER:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/1bd524e5631fc0bd116d15cf9fc62d07b5047f82/op_robot_tests/tests_files/aboveThreshold.robot#L8)):*
Done.

---

*[op_robot_tests/tests_files/aboveThreshold.robot, line 18 \[r3\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGMZGJVL1ViDSuwZKWR:-KGMrYQmQSF_XEvTY5ww:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/1bd524e5631fc0bd116d15cf9fc62d07b5047f82/op_robot_tests/tests_files/aboveThreshold.robot#L18)):*
Done.

---

*[op_robot_tests/tests_files/aboveThreshold.robot, line 24 \[r3\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGMZIjSK_nY4DEvDTol:-KGMrYrCV1XvKCRSgNHC:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/1bd524e5631fc0bd116d15cf9fc62d07b5047f82/op_robot_tests/tests_files/aboveThreshold.robot#L24)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/robot_tests/131)*

**Comment by selurvedu** [comment]
Conflicts!

**Comment by mykhaly** [comment]
Someone who will merge must resolve them :)

**Comment by Leits** [comment]
Reviewed 1 of 8 files at r1, 1 of 1 files at r15, 1 of 2 files at r16, 1 of 1 files at r18, 3 of 3 files at r19.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---

*[op_robot_tests/tests_files/aboveThreshold.robot, line 375 \[r19\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGWqE86-iFBjbQdfwCm:-KGWqE86-iFBjbQdfwCn:-1700436705) ([raw file](https://github.com/openprocurement/robot_tests/blob/ff39624d990008c7442f16b954f2af574f5a7dee/op_robot_tests/tests_files/aboveThreshold.robot#L375)):*
Ребейз видалив.

---

*[op_robot_tests/tests_files/service_keywords.py, line 424 \[r19\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGWm01V2ABcRyLLjwfI:-KGWm01Wy6BECtDBg6PT:741692071) ([raw file](https://github.com/openprocurement/robot_tests/blob/0f591737b853c6e12b087c50fbe04250a52a2e9f/op_robot_tests/tests_files/service_keywords.py#L424)):*
чому не ```if  complaintID == element['complaintID']```?

---

*[op_robot_tests/tests_files/service_keywords.py, line 435 \[r19\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGWmGo0Asl2MVmrKO9F:-KGWmGo0Asl2MVmrKO9G:-603362413) ([raw file](https://github.com/openprocurement/robot_tests/blob/0f591737b853c6e12b087c50fbe04250a52a2e9f/op_robot_tests/tests_files/service_keywords.py#L435)):*
Аналогічно ```if  document_id == element['title']```. Забери print-и.

---

*[op_robot_tests/tests_files/brokers/openprocurement_client.robot, line 434 \[r19\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGWq0lAq94TDYnMbuwk:-KGWq0lBhc2CI66Z1klY:-4901708) ([raw file](https://github.com/openprocurement/robot_tests/blob/0f591737b853c6e12b087c50fbe04250a52a2e9f/op_robot_tests/tests_files/brokers/openprocurement_client.robot#L434)):*
Тут потрібно логувати тендер?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/robot_tests/131)*

**Comment by mykhaly** [comment]
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---

*[op_robot_tests/tests_files/aboveThreshold.robot, line 375 \[r19\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGWqE86-iFBjbQdfwCm:-KGX3nxQ3XIF9wUvF3kg:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/ff39624d990008c7442f16b954f2af574f5a7dee/op_robot_tests/tests_files/aboveThreshold.robot#L375)):*
Done.

---

*[op_robot_tests/tests_files/service_keywords.py, line 424 \[r19\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGWm01V2ABcRyLLjwfI:-KGX3lxa9x9wHT8JAGjo:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/0f591737b853c6e12b087c50fbe04250a52a2e9f/op_robot_tests/tests_files/service_keywords.py#L424)):*
Done.

---

*[op_robot_tests/tests_files/service_keywords.py, line 435 \[r19\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGWmGo0Asl2MVmrKO9F:-KGX3ma4DYaQHFI395JX:-499146009) ([raw file](https://github.com/openprocurement/robot_tests/blob/0f591737b853c6e12b087c50fbe04250a52a2e9f/op_robot_tests/tests_files/service_keywords.py#L435)):*
Done.

---

*[op_robot_tests/tests_files/brokers/openprocurement_client.robot, line 434 \[r19\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/131#-KGWq0lAq94TDYnMbuwk:-KGWy4kZQkyu7fg5gcIB:1846725100) ([raw file](https://github.com/openprocurement/robot_tests/blob/0f591737b853c6e12b087c50fbe04250a52a2e9f/op_robot_tests/tests_files/brokers/openprocurement_client.robot#L434)):*
Це просто перенесений Log.  В майбутньому треба пройтися по всіх файлах і видалити всі логи.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/robot_tests/131)*

**Comment by Leits** [comment]
Reviewed 1 of 1 files at r22, 1 of 2 files at r23, 1 of 1 files at r25, 3 of 3 files at r26.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/robot_tests/131)*

**Pull request merged**